packages/terra-spacer/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "terra-spacer",
+  "main": "lib/Spacer.js",
+  "version": "0.0.0",
+  "description": "{insert description}",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-spacer",
+    "Spacer",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "terra-base": "^2.6.1"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "react-bps": "0.0.4",
+    "terra-base": "^1.0.0",
+    "terra-toolkit": "^2.1.0"
+  },
+  "scripts": {
```

Comment:
These scripts need update to be consistent with our current setup. See [here](https://github.com/cerner/terra-core/blob/master/packages/terra-list/package.json#L39) for an example.

Reply:
Updated 

1)

https://github.com/cerner/terra-core/pull/1011/commits/0a85758b3db9c076cef2ef5988e96c1b90a2d35a#diff-f3bfa5e0bf12fb4af2128aaf883cc625R40

2)

https://github.com/cerner/terra-core/commit/51079e4e1f7a6e3bb384bba084478bb7fcf12335#diff-f3bfa5e0bf12fb4af2128aaf883cc625R34